src/nbl/ext/MitsubaLoader/CMitsubaMaterialCompilerFrontend.cpp

```diff
@@ -88,7 +88,9 @@ CMitsubaMaterialCompilerFrontend::EmitterNode* CMitsubaMaterialCompilerFrontend:
         auto [image, sampler, meta] = getEmissionProfile(_emitter->area.emissionProfile);
         if (image) {
             profile.texture = { image, sampler, 1.f };
-            auto inverseTransform = core::transpose(core::concatenateBFollowedByA(transform, _emitter->transform.matrix));
+            auto inverseTransform = core::concatenateBFollowedByA(transform, _emitter->transform.matrix);
```

Comment:
why do you call it an inverse transform though ? its not an inverse of anything really, its just an IES space to World Space transform

Reply:
[resolved](https://github.com/Devsh-Graphics-Programming/Nabla/commit/6d595f64d75bfd286bd3b91ca8a30d0ee27403dd)